infrastructure/templates/public-api/main.bicepparam

```diff
@@ -0,0 +1,116 @@
+param deploymentRepo = 'N/A'
+
+//Networking Params --------------------------------------------------------------------------
+@description('Networking : Deploy subnets for networking')
+param deploySubnets = true
+
+@description('Networking : Included whitelist')
+param storageFirewallRules = ['0.0.0.0/0']
+
+//Storage Params ------------------------------------------------------------------------------
+@description('Storage : Name of the root fileshare folder name')
+@minLength(3)
+@maxLength(63)
+param fileShareName = 'data'
+
+@description('Storage : Type of the file share quota')
+param fileShareQuota = 1
+
+//PostgreSQL Database Params -------------------------------------------------------------------
+@description('Database : administrator login name')
+@minLength(1)
+param dbAdministratorLoginName = 'PostgreSQLAdmin'
+
+@description('Database : administrator password')
+@minLength(8)
+@secure()
+param dbAdministratorLoginPassword = ''
+
+@description('Database : Azure Database for PostgreSQL sku name ')
+@allowed([
+  'Standard_B1ms'
+  'Standard_D4ads_v5'
+  'Standard_E4ads_v5'
+])
+param skuName = 'Standard_B1ms'
+
+@description('Database : Azure Database for PostgreSQL Storage Size in GB')
+param storageSizeGB = 32
+
+@description('Database : Azure Database for PostgreSQL Autogrow setting')
+param autoGrowStatus = 'Disabled'
+
+//Container Registry Params ----------------------------------------------------------------
+@minLength(5)
+@maxLength(50)
+@description('Registry : Name of the azure container registry (must be globally unique)')
+param containerRegistryName = 'eesapiacr'
+
+//Container App Params
+@description('Container App : Specifies the container image to deploy from the registry <name>:<tag>.')
+param acrHostedImageName = 'hello-world'
+
+@description('Specifies the container port.')
+param targetPort = 80
+
+@description('Container App : Specifies the container image to seed to the ACR.')
+param containerSeedImage = 'mcr.microsoft.com/azuredocs/aci-helloworld'
+
+@description('Select if you want to seed the ACR with a base image.')
+param seedRegistry = true
+
+//ServiceBus Queue Params -------------------------------------------------------------------
+@description('Name of the Service Bus namespace')
+param namespaceName = 'etlnamespace'
+
+@description('Name of the Queue')
+param queueName = 'etlfunctionqueue'
+
+//ETL Function Paramenters ------------------------------------------------------------------
+@description('Application : Insights name')
+param applicationInsightsName = 'etlFunctionInsights'
+
+@description('Function App Plan operating system')
+@allowed([
+  'Windows'
+  'Linux'
+])
+param appServiceplanOS = 'Linux'
+
+@description('Function App : Runtime Language')
+@allowed([
+  'dotnet'
+  'node'
+  'python'
+  'java'
+])
+param functionAppRuntime = 'python'
+
```

Comment:
Would get rid of all of these descriptions. They're defined in `main.bicep` so there's no need to redeclare them. Otherwise, we're going to quickly see discrepancies between the two as people change things.